mibs/deltanet/deltanet.mib

```diff
@@ -0,0 +1,19 @@
+Deltanet-MIB DEFINITIONS ::= BEGIN
```

Comment:
Hi @Bierchermuesli 
Could you please rename the file to `Deltanet-MIB`. We need to have all MIBs named after their official definition for readability and deduplication ... 
Thanx